ramses_rf/devices_hvac.py

```diff
@@ -251,44 +251,102 @@ class HvacVentilator(DeviceHvac):  # FAN: RP/31DA, I/31D[9A]
     _SLUG: str = DEV_TYPE.FAN
 
     @property
-    def boost_timer(self) -> Optional[int]:
-        return self._msg_value(_31DA, key="remaining_time")
+    def exhaust_fan_speed(self) -> Optional[float]:
+        return self._msg_value((_31DA), key=EXHAUST_FAN_SPEED)
+        
+    @property
+    def remaining_time(self) -> Optional[int]:
+        return self._msg_value(_31DA, key=REMAINING_TIME)
 
     @property
     def co2_level(self) -> Optional[int]:
-        return self._msg_value(_31DA, key="co2_level")
+        return self._msg_value(_31DA, key=CO2_LEVEL)    
+
+    @property
+    def indoor_humidity(self) -> Optional[float]:
+        return self._msg_value(_31DA, key=INDOOR_HUMIDITY)
 
     @property
-    def fan_rate(self) -> Optional[float]:
-        return self._msg_value((_31D9, _31DA), key="exhaust_fan_speed")
```

Comment:
rename this as `exhaust_fan_speed` - see above.

Reply:
also - switch string literal fro the corresponding constant

Reply:
I removed this as the 31D9 does not always return exhaust fan speed. If I include it the parser should be adjusted to account for that